src/components/Proposals/DraftProposal/DraftEditForm.tsx

```diff
@@ -295,6 +382,10 @@ const DraftEditForm = forwardRef<DraftEditFormRef, DraftEditFormProps>(
                     formValues.proposalType === ProposalType.Tactical
                       ? formValues.quorumThreshold
                       : undefined,
+                  approvalThreshold:
+                    formValues.proposalType === ProposalType.Tactical
```

Comment:
Should this actually be: 

```suggestion
                    formValues.proposalType !== ProposalType.Standard
```

since the other proposal types (e.g. super majority) also have approval thresholds set

Reply:
Sorry disregard this - realized that this is not a value that would be set at creation time since it's based on the total votes, so it has to be computed while the proposal is active.